Solutions/Azure Activity/Hunting Queries/Rare_Custom_Script_Extension.yaml

```diff
@@ -35,14 +34,14 @@ query: |
   let RareCustomScriptExecution =  CurrentCustomScriptExecution
```

Comment:
Value of properties is getting parsed multiple times.

Reply:
thanks @devikamehra . Parsing and storing Properties in a variable now.